packages/core/src/extractors/prompts.ts

```diff
@@ -0,0 +1,52 @@
+export const defaultKeywordExtractorPromptTemplate = ({
+  context_str,
+  keywords,
+}: {
+  context_str: string;
+  keywords: number;
+}) => `
+  /${context_str}. Give ${keywords} unique keywords for this \
+  document. Format as comma separated. Keywords:
+`;
+
+export const defaultTitleExtractorPromptTemplate = ({
+  context_str,
+}: {
+  context_str: string;
+}) => `
+  /${context_str}. Give a title that summarizes all of the unique entities, titles or themes found in the context. Title:
```

Comment:
I would try to camel case if possible (if there are inconsistencies we can fix them). Also, if you look at the other prompts I've created, I think it's useful to create an "implied type" for each prompt so that the user can easily figure out what variables they need to handle if they're customizing the prompt.

Reply:
Yes, we are using camelCase on almost all, just a nit, thanks for pointing out